tests/functional/external_mode/test_virtualization_storageclass.py

```
@@ -64,20 +66,18 @@ def setup_and_teardown_cnv(self):
 
-        # --- TEARDOWN ---
-        # FIX: Only run uninstallation if CNV was not already active prior to execution
         if not self.cnv_pre_installed:
             log.info("Test finished. Initiating CNV Uninstallation (Teardown)...")
             try:
                 cnv_installer.uninstall_cnv(check_cnv_installed=True)
                 log.info("CNV uninstallation completed successfully.")
-            except Exception as e:
-                log.error(f"Failed to uninstall CNV during teardown: {str(e)}")
+            except CommandFailed as e:
+                log.error("Failed to uninstall CNV during teardown: %s", str(e))
```

Ruff (0.15.20)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)